orchagent/policerorch.cpp

```diff
@@ -219,7 +496,7 @@ void PolicerOrch::doTask(Consumer &consumer)
 
                 sai_object_id_t policer_id;
                 sai_status_t status = sai_policer_api->create_policer(
-                    &policer_id, gSwitchId, (uint32_t)attrs.size(), attrs.data());
+                        &policer_id, gSwitchId, (uint32_t)attrs.size(), attrs.data());
```

Comment:
can you check if this is tabspace, if so please fix

Reply:
This is because of auto-indent. fixed.